AnimView/th.cpp

```diff
@@ -34,7 +34,7 @@ SOFTWARE.
 #include <stdexcept>
 #include <vector>
 
-#include "../common/rnc.h"
+#include "../libs/rnc/rnc.h"
```

Comment:
Wouldn't it be nicer if source code doesn't have full paths to other code?

That is, do `#include "rnc/rnc.h"` here, and extend the include search path of the compiler with `-I../libs`. In that way, source code doesn't change if you move libraries or other code around in the project tree.

Reply:
Yes, I might be able to do that.